test/testkit/MockitoSugar.scala

```diff
@@ -23,8 +23,8 @@ import scala.concurrent.Future
 
 trait MockitoSugar extends org.scalatest.mock.MockitoSugar {
   // Unfortunately we cannot use the `when().thenReturn` format for spies.
-  def doReturnAsync() = Mockito.doReturn(Future.successful(()))
-  def doReturnAsync[A](value: A) = Mockito.doReturn(Future.successful(value))
+  def doReturnAsync() = Mockito.doReturn(Future.successful(()), Nil.toSeq:_*)
```

Comment:
That is rather weird! First I considered that .toSeq seems redundant, and can be removed. Then I dug a bit deeper and considered it a compiler bug until discovered the discussion here http://stackoverflow.com/questions/13358705/force-single-argument-in-scala-varargs/14290755#14290755